pkg/rtc/participant.go

```diff
@@ -1257,6 +1260,8 @@ func (p *ParticipantImpl) downTracksRTCPWorker() {
 			pkts = pkts[:0]
 			batchSize = 0
 		}
+
+		time.Sleep(5 * time.Second)
```

Comment:
Moving the sleep down here to ensure first report(s) go out as soon as connection is established.